js/exporter/jspdf/export_data_grid_2.js

```diff
@@ -48,7 +48,28 @@ function exportDataGrid(doc, dataGrid, options) {
                             pdfCell.colSpan = cellMerging.colspan;
                         }
                     } else if(rowType === 'group') {
-                        pdfCell.colSpan = columns.length - 1;
+                        if(cellData.cellSourceData.groupSummaryItems?.length > 0) {
+                            pdfCell._summaryItem = true;
+                        }
+
+                        if(cellIndex > 0) {
+                            const prevCellsHaveSummaryItems = currentRow.slice(1).filter(pdfCell => pdfCell._summaryItem).length > 0;
+                            if(!pdfCell._summaryItem && !prevCellsHaveSummaryItems) {
```

Comment:
'_summaryItem' is used in this method only but declared in a long living object.
Can we change it to a local variable?

Reply:
Ok, we can remove the '_summaryItem' field and use the 'isDefined(pdfCell.text)' expression instead. In this case we will assume that if the text is defined in pdfCell of the group row, then this is a summary item aligned by column. Except the first cell.

Reply:
Done